api/internal/core/store/validate.go

```diff
@@ -125,17 +125,17 @@ func (v *APISIXJsonSchemaValidator) Validate(obj interface{}) error {
 		for pluginName, pluginConf := range plugins {
 			schemaDef := conf.Schema.Get("plugins." + pluginName).String()
 			if schemaDef == "" {
-				return fmt.Errorf("schema not found")
+				return fmt.Errorf("scheme validate failed: schema not found")
 			}
 
 			s, err := gojsonschema.NewSchema(gojsonschema.NewStringLoader(schemaDef))
 			if err != nil {
-				return fmt.Errorf("new schema failed: %w", err)
+				return fmt.Errorf("scheme validate failed: %w", err)
 			}
 
 			ret, err := s.Validate(gojsonschema.NewGoLoader(pluginConf))
 			if err != nil {
-				return fmt.Errorf("validate failed: %w", err)
+				return fmt.Errorf("scheme validate failed: %w", err)
```

Comment:
I found that the same error messages repeatedly appeared in the files, would it be better to  manage error messages in one place just like [error.go](https://github.com/apache/apisix-dashboard/blob/master/api/errno/error.go) before :)?

Reply:
Agree, but we can not do it in this version for now. When the first rc version is released, we will optimize it in subsequent versions

Reply:
+1 @ShiningRush